app/routes.py

```diff
@@ -1,2 +1,212 @@
+    db.session.commit()
+    return make_response(jsonify({"id": customer.customer_id})) 
+
+@videos_bp.route("", methods = ["GET"])
+def get_videos():
+    videos = Video.query.all()
+    videos_response = []
+    if videos is None:
+        return []
+    for video in videos:
+        videos_response.append(video_response_helper(video))
+    return jsonify(videos_response)
+
+@videos_bp.route("/<video_id>", methods = ["GET"])
+def get_one_video(video_id):
+    video = Video.query.get_or_404(video_id)
+    return video_response_helper(video)
+
+@videos_bp.route("", methods = ["POST"])
+def post_videos():
+    request_body = request.get_json()
+    if "title" not in request_body or "release_date" not in request_body or "total_inventory" not in request_body:
+        return make_response({"details": "Invalid data"}, 400)
+    else:
+        new_video = Video(title=request_body["title"], release_date=request_body["release_date"], total_inventory=request_body["total_inventory"])
+        db.session.add(new_video)
+        db.session.commit()
+        return make_response({
+            "id": new_video.video_id
+        }, 201)
+
+@videos_bp.route("/<video_id>", methods = ["PUT"])
+def update_videos(video_id):
+    video = Video.query.get(video_id)
+    if video is None:
+            return make_response({"details": "Video not found"}, 404)
+    form_data = request.get_json()
+    if "title" not in form_data or "release_date" not in form_data or "total_inventory" not in form_data:
+        return make_response({"details": "Invalid data"}, 400)
+    else:
+        video.title = form_data["title"]
+        video.release_date = form_data["release_date"]
+        video.total_inventory = form_data["total_inventory"]
+        db.session.commit()
+        return video_response_helper(video)
+
+@videos_bp.route("/<video_id>", methods = ["DELETE"])
+def delete_video(video_id):
+    video = Video.query.get(video_id)
+    if video is None:
+            return make_response({"details": "Video not found"}, 404)
+    else:
+        db.session.delete(video)
+        db.session.commit()
+        return make_response(jsonify({"id": video.video_id})) 
+
+@rentals_bp.route("/check-out", methods = ["POST"])
+def post_rentals_check_out():
+    request_body = request.get_json()
+    if "customer_id" not in request_body or "video_id" not in request_body:
+        return make_response({"details": "Invalid data"}, 404)
+    elif type(request_body["customer_id"]) is not int :
+        return make_response({"details": "Invalid customer_id"}, 400)
+    elif type(request_body["video_id"]) is not int :
+        return make_response({"details": "Invalid video_id"}, 400)
+    new_rental = Rental(customer_id=request_body["customer_id"], video_id=request_body["video_id"])
+    customer = Customer.query.get(new_rental.customer_id)
+
+    video = Video.query.get(new_rental.video_id)
+    # customer.videos_checked_out_count += 1
+    # video.available_inventory -=1
```

Comment:
don't forget to get rid of code you won't be using when you submit final projects!

```suggestion

```